src/Widgets/ReleaseRow.vala

```diff
@@ -29,8 +29,17 @@ public class AppCenter.Widgets.ReleaseRow : Gtk.Box {
     }
 
     construct {
-        var header_icon = new Gtk.Image.from_icon_name ("tag-symbolic");
+        orientation = Gtk.Orientation.VERTICAL;
+        spacing = 6;
+        margin_bottom = 6;
+
+        create_header ();
+        create_description ();
+        create_issues ();
```

Comment:
We prefer to to not split the constructor into multiple methods.

Reply:
Got it :+1: Just set removed these methods and put it back to the constructor